app/src/main/java/org/mozilla/tv/firefox/webrender/WebRenderFragment.kt

```diff
@@ -135,8 +135,13 @@ class WebRenderFragment : EngineViewLifecycleFragment(), Session.Observer {
          *  memory leak while clearing data. See [WebViewCache.clear] as well as #1720
          */
         serviceLocator?.screenController?.currentActiveScreen?.observe(viewLifecycleOwner, Observer {
-            if (it != ScreenControllerStateMachine.ActiveScreen.WEB_RENDER) {
-                engineView.pauseAllVideoPlaybacks()
+            when (it) {
```

Comment:
nit: I think this is more readable as an if statement because the comparison (`it == WEB_RENDER`) is on a single line.